hedera-node/hedera-app/src/main/java/com/hedera/node/app/version/HederaSoftwareVersion.java

```diff
@@ -77,17 +75,12 @@ public HederaSoftwareVersion(
         this.servicesVersion = Objects.requireNonNull(servicesVersion, "servicesVersion must not be null");
     }
 
-    /**
```

Comment:
Given the fact that `ServicesSoftwareVersion` replaces `HederaSoftwareVersion`, should `HederaSoftwareVersion` be deprecated?